src/main/java/com/lpvs/entity/auth/OAuthAttributes.java

```diff
@@ -55,7 +56,8 @@ public enum OAuthAttributes {
         this.of = of;
     }
 
-    public static MemberProfile extract(String registrationId, Map<String, Object> attributes) {
+    public static MemberProfile 
+    extract(String registrationId, Map<String, Object> attributes) {
```

Comment:
Doesn't seem to be a substantial change.

Reply:
thanks, good point.